src/js/app.js

```diff
@@ -1,36 +1,54 @@
+{ "chainId": 1285, "assetAddress": "0x1b30a3b5744e733d8d2f19f0812e3f79152a8777", "tokenId": 2, "name": "stone" },
+{ "chainId": 1285, "assetAddress": "0x1b30a3b5744e733d8d2f19f0812e3f79152a8777", "tokenId": 3, "name": "iron" },
+{ "chainId": 1285, "assetAddress": "0x1b30a3b5744e733d8d2f19f0812e3f79152a8777", "tokenId": 4, "name": "gold" },
+{ "chainId": 1285, "assetAddress": "0x1b30a3b5744e733d8d2f19f0812e3f79152a8777", "tokenId": 5, "name": "experience" },
+{ "chainId": 1285, "assetAddress": "0x1b30a3b5744e733d8d2f19f0812e3f79152a8777", "tokenId": 10, "name": "grain" },
+{ "chainId": 1285, "assetAddress": "0x1b30a3b5744e733d8d2f19f0812e3f79152a8777", "tokenId": 12, "name": "string" },
+{ "chainId": 1285, "assetAddress": "0x1b30a3b5744e733d8d2f19f0812e3f79152a8777", "tokenId": 13, "name": "fish_specimen" },
+{ "chainId": 1285, "assetAddress": "0x1b30a3b5744e733d8d2f19f0812e3f79152a8777", "tokenId": 16, "name": "moonstone" },
 
+{ "chainId": 2109, "assetAddress": "0x95370351df734b6a712ba18848b47574d3e90e61", "tokenId": 1, "name": "pumpkins" },
+{ "chainId": 2109, "assetAddress": "0x95370351df734b6a712ba18848b47574d3e90e61", "tokenId": 2, "name": "blood_crystals" },
+{ "chainId": 2109, "assetAddress": "0x95370351df734b6a712ba18848b47574d3e90e61", "tokenId": 3, "name": "dna" },
+{ "chainId": 2109, "assetAddress": "0x95370351df734b6a712ba18848b47574d3e90e61", "tokenId": 4, "name": "mobidium" },
+{ "chainId": 2109, "assetAddress": "0x95370351df734b6a712ba18848b47574d3e90e61", "tokenId": 5, "name": "wood" },
+{ "chainId": 2109, "assetAddress": "0x95370351df734b6a712ba18848b47574d3e90e61", "tokenId": 6, "name": "stone" },
+{ "chainId": 2109, "assetAddress": "0x95370351df734b6a712ba18848b47574d3e90e61", "tokenId": 7, "name": "iron" },
+{ "chainId": 2109, "assetAddress": "0x95370351df734b6a712ba18848b47574d3e90e61", "tokenId": 8, "name": "gold" }
+]
 
 /**
  * @typedef {Object} MaterialResult
- * @property {number} tokenId - id of token
+ * @property {number} chainId
  * @property {string} name - name of token
+ * @property {number} tokenId - id of token
  * @property {number} lowestSell
  * @property {number} highestBuy
  */
 
 /**
- * Returns prices, tokenIds, and names for all minecraft NFTs
+ * Returns prices, tokenIds, chainIds, and names for all game NFTs
  * @returns {Promise<MaterialResult[]>}
  */
-async function getAllMaterialsMovr(){
-  const proms = []
-  let rawResult = [];
-  for(const material of materials){
-    const prom = getMaterialMovr(material.tokenId)
-    proms.push(prom)
-    prom.then(({lowestSell, highestBuy})=>{
-      rawResult.push({tokenId: material.tokenId, name: material.name, lowestSell, highestBuy})
-    })
-  }
-  await Promise.all(proms)
-  rawResult = rawResult.sort((a, b) => a.tokenId - b.tokenId)
+async function getAllMaterialsPrice(){
+
+  const results = await Promise.all(materials.map(async ({chainId, name, tokenId, assetAddress}) => {
```

Comment:
in my pull i have khaos integrated. i pass in the erc20 address as i believe that is what is needed to navigate pairs in their subgraph, so perhaps add in another field with the erc20 addresses? 

This is what I have: const materials = [{"tokenId": 1, "name":"pumpkins", "address": "0xb296335B6d4AB9F5e5Dc4bf115621c14ab70136C"}, {"tokenId": 2, "name":"blood_crystals", "address": "0x76ba6821A834E2d30A699312829f8204830CCF5B".toLowerCase()}, {"tokenId": 3, "name":"dna", "address": "0xff6ca2a78E4Dd4871256cc5B63dC8dbf02a36F3D".toLowerCase()}, {"tokenId": 4, "name":"mobidium", "address": "0xa1caaaa65fd4e144c4650f87933705b9a065008e"}, {"tokenId": 5, "name":"wood", "address": "0x0528Ae6c997BA4FfC4d6ADD898a2Cc9f26d4a872".toLowerCase()}, {"tokenId": 6, "name":"stone", "address": "0x122EBE2B679cF54Bc8a6e89c1009714b354e2d10".toLowerCase()}, {"tokenId": 7, "name":"iron", "address": "0x829191FEC0f25a27c5bDCdBbD0e37E6342b39945".toLowerCase()}, {"tokenId": 8, "name":"gold", "address": "0x549ab43056E3489335b47927FFcd4825ce484bA3".toLowerCase()}]

The lowercase thing is just a hacky thing to get results for queries to khaos subgraph (some case issues resulted in no results returned)